archive-press.php

```diff
@@ -0,0 +1,318 @@
+<?php
+get_header();
+
+$current_year_id = IGV_get_option('_igv_site_options', '_igv_current_fair_year');
+$current_year = (!empty($current_year_id)) ? get_term($current_year_id)->slug : false; 
+// if we have the term ID, get the slug, 
+// otherwise set it false for later conditionals
+
+$press_page_text = IGV_get_option('_igv_page_options', '_igv_press_page_text');
+$press_page_image_id = IGV_get_option('_igv_page_options', '_igv_press_page_image_id');
+$press_accreditation = IGV_get_option('_igv_page_options', '_igv_press_accreditation_text');
+
+?>
+
+<!-- main content -->
+
+<main id="main-content">
+
+  <!-- main posts loop -->
+  <section id="press-intro" class="section">
+    <div class="container">
+      <div class="row">
+        <div class="col col-l col-l-12">
+          <h1>
+          <?php 
+            _e('[:en]Press[:es]Prensa;'); 
+          ?>
+          </h1>
+        </div>
+      </div>
+<?php 
+if (!empty($press_page_text) || !empty($press_page_image_id)) { 
+?>
+      <div class="row">
+<?php 
+  if (!empty($press_page_image_id)) {
+?>
+        <div class="col col-l col-l-6">
+          <?php echo wp_get_attachment_image($press_page_image_id, 'col-6'); ?>
+        </div>
+<?php 
+  } if (!empty($press_page_text)) { 
+?>
+        <div class="col col-l col-l-6">
+          <div class="font-size-h3">
+            <?php echo apply_filters( 'the_content', $press_page_text ); ?>
+          </div>
+          <div class="font-size-h4">
+            <?php echo (!empty($press_accreditation) ? apply_filters( 'the_content', $press_accreditation ) : ''); ?>
```

Comment:
im really only using them where i have to validate a meta value, echo it if it exists, and echo nothing if not.  seems fine to me.